src/matlab2tikz.m

```diff
@@ -518,19 +531,19 @@ function matlab2tikz(varargin)
     % Finally print it to the file
     addComments(fid, m2t.content.comment);
     addStandalone(m2t, fid, 'preamble');
-    addCustomCode(fid, '', m2t.cmdOpts.Results.extraCode, '');
+    addCustomCode(fid, '', m2t.args.extraCode, '');
     addStandalone(m2t, fid, 'begin');
 
     printAll(m2t, m2t.content, fid); % actual plotting happens here
 
-    addCustomCode(fid, '\n', m2t.cmdOpts.Results.extraCodeAtEnd, '');
+    addCustomCode(fid, '\n', m2t.args.extraCodeAtEnd, '');
 
     addStandalone(m2t, fid, 'end');
 end
 % ==============================================================================
 function addStandalone(m2t, fid, part)
     % writes a part of a standalone LaTeX file definition
-    if m2t.cmdOpts.Results.standalone
+    if m2t.args.standalone
```

Comment:
So this is not related to the PR, but what do you think about allowing custom preambles/standalones?

Ipersonally have 3 matlab2tikz versions. One i use for my Thesis/Papers which has some modifications for the preamble that are necessary for my workflow, one for development and one for normal stuff/testing.

It would be awesome if we would add "CustomeStandalone" or whatever.

Thoughts?